src/active_stakers.rs

```diff
@@ -76,8 +83,22 @@ impl ActiveStakers {
         pubkeys
     }
 
-    pub fn leader_schedule(&self) -> LeaderSchedule {
-        LeaderSchedule::new(self.pubkeys())
+    pub fn leader_schedule(&self, bank: &Bank, len: u64) -> LeaderSchedule {
```

Comment:
No need to pass in a whole bank here, just a `seed: &Pubkey`.

Reply:
In fact, better than a pubkey, how about a 32-byte array? Give the caller the option to seed it differently.